projects/Mallard/src/theme/typography.ts

```diff
@@ -1,6 +1,9 @@
 const families = {
-    sans: 'GuardianTextSans-Regular',
-    text: 'GuardianTextEgyptian-Reg',
+    sans: {
+        regular: 'GuardianTextSans-Regular',
+        bold: 'GuardianTextSans-Bold',
+    },
+    text: { regular: 'GuardianTextEgyptian-Reg' },
```

Comment:
Is it worth adding `regular` in here while it's there?

Reply:
you mean bold, right? totes! I wanna do a huge fonts pr later today